### PR TITLE
Add posh-dotnet

### PR DESCRIPTION
**Pull request by bergmeister**
[posh-dotnet](https://github.com/bergmeister/posh-dotnet) adds `PowerShell` tab completion for the [dotnet CLI](https://github.com/dotnet/cli) (for version 1 and  2 with version 2 being much more enhanced)

### CODE REVIEW

**Comment by thangchung** [comment]
@bergmeister Thank you for your PR, but this repository only accepts the repository that was created at least 3 months. I have seen that your repo only created recently, right?

**Comment by bergmeister** [comment]
Yes, it is relatively new but I can wait if you want to.

File: README.md [hunk 1]
```diff
@@ -433,6 +433,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [letsencrypt-win-simple](https://github.com/Lone-Coder/letsencrypt-win-simple) - A Simple ACME Client for Windows.
 * [OctoLinker](https://github.com/OctoLinker/browser-extension) - Navigate through `projects.json` files efficiently with the OctoLinker browser extension for GitHub.
 * [Opserver](https://github.com/opserver/Opserver) - Stack Exchange's Monitoring System.
+* [posh-dotnet](https://github.com/bergmeister/posh-dotnet) - `PowerShell` tab completion for the [dotnet CLI](https://github.com/dotnet/cli).
```
Comment (ngohungphuc) [inline README.md]: @bergmeister please resolve conflict then I will merge your pr. Thank you

Comment (bergmeister) [reply README.md]: @ngohungphuc Thanks. The merge conflict is resolved now. It was a trival whitespace or carriage return conflict that TortoiseGit could auto-resolve for me.

**Comment by ngohungphuc** [comment]
@bergmeister one quick question does this cli support .Net Core

**Comment by ngohungphuc** [comment]
thank you. merge now

**Pull request merged**